packages/clay-pagination/src/PaginationWithBasicItems.tsx

```diff
@@ -164,6 +164,7 @@ const ClayPaginationWithBasicItems = React.forwardRef<HTMLUListElement, IProps>(
 					disabled={internalActive === 1}
 					href={previousHref}
 					onClick={() => setActive(previousPage)}
+					role={internalActive === 1 ? undefined : 'button'}
```

Comment:
In both these cases shouldn't we be checking if the href is actually empty or null? Don't we sometimes use an href for going to the previous or next page, in which case the role should stay a link and not a button, right?

Reply:
Good point, I'll update it